jobs/testable.rb

```diff
@@ -0,0 +1,27 @@
+require 'dashing'
+require File.expand_path('../../lib/travis_backend', __FILE__)
+
+SCHEDULER.every '1d', first_in: '1s' do |_job|
+  travis_backend = TravisBackend.new
+  github_backend = GithubBackend.new
+
+  testable_repos = github_backend.testable_repos
+
+  travis_repos = testable_repos.map { |repo| travis_backend.get_repo(repo) }
+  repos_without_builds = travis_repos.select { |repo| repo['last_build_id'].nil? }
+  ignored_repos = ENV['TRAVIS_IGNORED_REPOS'].split(',').compact
+
+  items = repos_without_builds.map do |repo|
```

Comment:
So here, we're only filtering out the "testable" repos that _don't_ have builds? In a file called `testable.rb` with no function names or other forms of documentation, I'd expect it to return all testable repos, or return a set of [testable-with-builds, testable-without-builds].